recipes/pyg-lib/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -ex
+
+if [[ ${cuda_compiler_version} != "None" ]]; then
+  export FORCE_CUDA=1
+  export CUDA_ARCH_LIST=75
```

Comment:
Please target the same CUDA architectures as pytorch since you are linking/depending on it. Compiling for only 75 would exclude a lot of users.

https://github.com/conda-forge/pytorch-cpu-feedstock/blob/main/recipe/build_pytorch.sh#L94-L111

Reply:
Thanks. I was simply using the upstream recommendation.

Maybe I can simply not provide any `CUDA_ARCH_LIST` variable instead of hard-coding it from your above example.

I will remove it. Please let me know if it's ok or if you prefer me to hard-code those.

Reply:
Thanks for the context here. It's good to know about this CMake feature.